test/docker-osx-dev.bats

```diff
@@ -417,6 +417,12 @@ export NEW_ENV_VARIABLE_2=VALUE2"
   assert_output '/some/path'
 }
 
+@test "find_path_to_sync_parent should match paths with weird characters" {
+  export PATHS_TO_SYNC='/some/path2()$HI'
```

Comment:
#102 does look suspiciously similar.

On Fri, Nov 13, 2015, 1:49 PM Yevgeniy Brikman <notifications@github.com>
wrote:

> In test/docker-osx-dev.bats
> <https://github.com/brikis98/docker-osx-dev/pull/143#discussion_r44838307>
> :
>
> > @@ -417,6 +417,12 @@ export NEW_ENV_VARIABLE_2=VALUE2"
> >    assert_output '/some/path'
> >  }
> >
> > +@test "find_path_to_sync_parent should match paths with weird characters" {
> > +  export PATHS_TO_SYNC='/some/path2()$HI'
>
> Ah, I just realized that code does an eval. I can't remember why that's
> there instead of just calling greadlink directly on $path...
>
> Also, is #102 <https://github.com/brikis98/docker-osx-dev/issues/102> the
> same issue?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brikis98/docker-osx-dev/pull/143/files#r44838307>.
>